gps/app/locales/en.json

```diff
@@ -5,22 +5,18 @@
     "operating_system_abbr": "OS:",
```

Comment:
this file has just been copied from `1.0.65`

Reply:
This is probably not quite enough. keys likely changed a lot.

Do a `yarn i18n:extract` and you'll see it add/remove keys based on all keys found in the js views.

Anything that is added/removed here will be added/removed from the translation mgmt system Lokalise. So we should be extra careful here before submitting.